src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs

```diff
@@ -216,5 +222,16 @@ private void OnInCanvasSearchKeyDown(object sender, KeyEventArgs e)
 
             return nextselectedMemberIndex;
         }
+
+        private void OnMembersListBoxMouseWheel(object sender, MouseWheelEventArgs e)
+        {
+            var listBox = sender as FrameworkElement;
+
+            var scrollViewer = listBox.ChildOfType<ScrollViewer>();
```

Comment:
There's no `null` checks for both `listBox` and `scrollViewer`, can you add those? Thanks!